### PR TITLE
Starting FAQ section

### PR DESCRIPTION
**Pull request by andyfleming**
Adding answers about `spinnaker-local.yml` and slack notification configuration.

### CODE REVIEW

File: documentation/faq.md [hunk 1]
```diff
@@ -10,6 +10,18 @@ lang: en
...
 
-To be populated with initial content by November 20, 2015.
+* [Where do I find `spinnaker-local.yml`?](#where-do-i-find-spinnaker-local-yml)
+* [How do I set up Slack notificaitons for a pipeline?](#how-do-i-set-up-notificaitons-for-a-pipeline)
 
+### Where do I find `spinnaker-local.yml`?
+
+You can find it in `/opt/spinnaker/config/`.
+
+### How do I set up Slack notificaitons for a pipeline?
```
Comment (duftler) [inline documentation/faq.md]: s/notificaitons/notifications

File: documentation/faq.md [hunk 2]
```diff
@@ -10,6 +10,18 @@ lang: en
 
 # Frequently Asked Questions
 
-To be populated with initial content by November 20, 2015.
+* [Where do I find `spinnaker-local.yml`?](#where-do-i-find-spinnaker-local-yml)
+* [How do I set up Slack notificaitons for a pipeline?](#how-do-i-set-up-notificaitons-for-a-pipeline)
```
Comment (duftler) [inline documentation/faq.md]: s/notificaitons/notifications in both spots

File: documentation/faq.md [hunk 3]
```diff
@@ -10,6 +10,18 @@ lang: en
...
 
+### Where do I find `spinnaker-local.yml`?
+
+You can find it in `/opt/spinnaker/config/`.
+
+### How do I set up Slack notificaitons for a pipeline?
+
+1. Maker sure Spinnaker is up to date.
```
Comment (duftler) [inline documentation/faq.md]: s/Maker/Make

Comment (duftler) [inline documentation/faq.md]: s/up to date/up-to-date

File: documentation/faq.md [hunk 4]
```diff
@@ -10,6 +10,18 @@ lang: en
...
+2. Add Slack credentials in `spinnaker-local.yml`.
+3. Configure notifications
+  * Option A) Add a notification to a single pipeline in the "configure" view.
+  * Option B) Enable notifications for "any"/all pipelines for an applicaiton under `/#/applications/your-application/config` where it says "Add Notification Preferences".
```
Comment (duftler) [inline documentation/faq.md]: s/applicaiton/application

Comment (tomaslin) [inline documentation/faq.md]: should say all pipelines for the application

**Comment by anotherchrisberry** [comment]
@duftler @tomaslin is this PR still relevant, or should it be closed?

**Comment by tomaslin** [comment]
I'm going to close this issue for now and incorporate the content into a separate FAQ PR

**Pull request closed without merging**